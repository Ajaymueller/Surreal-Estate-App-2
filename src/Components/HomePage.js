import React from "react";
import "../styles/HomePage.css";

const HomePage = () => {
    return (
      <div className="home-page">
        <aside className="home-page__welcome">
          <h2 className="home-page__heading">Make your dream home a reality</h2>
          <p>
            Find your Dream Home here with Surreal Estate 
          </p>
        </aside>
      </div>
    );
  };

  export default HomePage;