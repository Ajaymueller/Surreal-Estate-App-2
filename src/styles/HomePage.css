.home-page {
    display: flex;
    flex-wrap: wrap;
    flex-direction: column;
    background-image: url("images/houses-background.jpeg");
    background-repeat: no-repeat;
    background-size: cover;
    height: 100vh;
}

.home-page h2 {
    font-size: 100px;
    text-align: center;
}

.home-page p {
    font-size: xx-large;
    text-align: center;
}