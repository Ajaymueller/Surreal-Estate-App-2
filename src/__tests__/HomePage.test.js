import React from 'react';
import { render } from '@testing-library/react';
import HomePage from '../Components/HomePage';
import { MemoryRouter } from 'react-router-dom';
import '@testing-library/jest-dom/extend-expect';

describe("HomePage", () => {
    it("component renders correctly", () => {
    const { asFragment } = render(
    <MemoryRouter><HomePage /></MemoryRouter> )
    expect(asFragment).toMatchSnapshot();
    });
});