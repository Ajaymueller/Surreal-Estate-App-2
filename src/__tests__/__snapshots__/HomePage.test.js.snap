exports[`HomePage component renders correctly 1`] = `[Function]`;
